Extension/src/LanguageServer/extension.ts

```diff
@@ -1162,3 +1162,39 @@ export function UpdateInsidersAccess(): void {
         void vscode.commands.executeCommand("workbench.extensions.installExtension", "ms-vscode.cpptools", { installPreReleaseVersion: true }).then(undefined, logAndReturn.undefined);
     }
 }
+
+export async function PreReleaseCheck(): Promise<void> {
+    // First we need to make sure the user isn't already on a pre-release version.
+    if (!util.packageJson.__metadata?.preRelease === true) {
```

Comment:
We have a function in common.ts called `getCppToolsTargetPopulation()` that we usually use to determine if the user is on the pre-release or not.

Reply:
Thanks! The terms of "Insiders" and "Pre-Release" are new to me and I was a little confused if they were the same or not, and in which scenario.

Reply:
Sorry about that.  "Insiders" is the old name, "Pre-release" is the new name, but they represent the same thing.